configs/dev.js

```diff
@@ -1,12 +1,13 @@
 const {
-  DB_HOST, DB_PORT, DB_USER, DB_PWD,
+  DB_HOST, DB_PORT, DB_USER, DB_PWD, SECRET_KEY,
 } = process.env;
 
 const DB_URL = `mongodb://${DB_HOST}:${DB_PORT}/easy-event-v2`;
 const devConfig = {
   DB_URL,
   DB_USER,
   DB_PWD,
+  SECRET_KEY,
```

Comment:
Create a file `configs/base.js` that contain those config from `.env`
--> Now it's duplicated

Reply:
Please check my new update and review again

Reply:
https://github.com/EasyEventV2/ee2_api/pull/1/commits/211a357d9c7cbf2f9d23f130d288f01545cdd4af